open_tab_tracker/Database.py

```diff
@@ -78,12 +92,44 @@ def print_dataframe(self, df: pd.DataFrame):
     def get_database_values_as_dataframe(self) -> pd.DataFrame:
         conn = sql.connect(self.database_file)
         cur = conn.cursor()
-        cur.execute("SELECT * FROM tab_count")
+        cur.execute("SELECT * FROM browser_tab_count")
         rows = cur.fetchall()
         conn.close()
-        df = pd.DataFrame(rows, columns=["datetime", "firefox_tab_count"])
+        df = pd.DataFrame(rows, columns=["datetime", "browser_type", "tab_count"])
         df["datetime"] = df["datetime"].apply(
             lambda x: self.convert_utc_datetime_to_local_formatted_string(x)
         )
-        df["firefox_tab_count"].astype(int)
+        df["tab_count"] = df["tab_count"].astype(int)
         return df
+
+    def migrate_database_if_needed(self):
+        """Migrate data from old table (tab_count) to new table (browser_tab_count)"""
+        conn = sql.connect(self.database_file)
+        cursor = conn.cursor()
+        
+        # Check if old table exists
+        cursor.execute("SELECT name FROM sqlite_master WHERE type='table' AND name='tab_count'")
+        old_table_exists = cursor.fetchone() is not None
+        
+        if old_table_exists:
+            logger.info("Found old table 'tab_count', migrating data...")
+            
+            # Check if data was already migrated by looking for any records in new table
+            cursor.execute("SELECT COUNT(*) FROM browser_tab_count")
+            new_table_count = cursor.fetchone()[0]
+            
+            if new_table_count == 0:
```

Comment:
What do you think about prompting the user to delete the old table? Or just flat out deleting it if we have migrated successfully?

We can check the number of rows and see if they match what's expected. If yes, let's reap the old table? No need to hold onto them.